oak/server/storage/storage_node.h

```diff
@@ -32,9 +36,21 @@ class StorageNode final : public NodeThread {
   StorageNode(const std::string& name, const std::string& storage_address);
 
  private:
+  const asylo::StatusOr<std::string> Encrypt(asylo::AesGcmSivCryptor* cryptor,
+                                             const absl::string_view& data);
+  const asylo::StatusOr<std::string> EncryptDatumName(const absl::string_view& datum_name);
+  const asylo::StatusOr<std::string> EncryptDatumValue(const absl::string_view& datum_value);
+
+  const asylo::StatusOr<std::string> DecryptDatumValue(const absl::string_view& datum_value);
+
   void Run() override;
 
+  FixedNonceGenerator* fixed_nonce_generator_;
```

Comment:
Could the ownership of this be made explicit, either by having `FixedNonceGenerator fixed_nonce_generator_;` or `std::unique_ptr<FixedNonceGenerator> fixed_nonce_generator_`?  At the moment, I *think* this will leak on storage node termination.

Reply:
Clarified ownership.